### ISSUE TITLE
[1][ahdigital][3] Typeahead for new Send modal

### ISSUE THREAD

**Issue by bangzi1001** [bitshares/bitshares-ui#1330]
There are lot of Air Drop coins in BTS recently. We can hide the coins in Portfolio page, but when we do Transfer, it still shown under the list of coins. Imagine we have over 100 Air Drop Coins and we need to find BTS from 100 Coins in drop down list. OMG

**Comment by wmbutler** [comment]
Good point. Send modal needs to use the typeahead. UX needed. Don't change the legacy transfer since it's going away.

![screen shot 2018-03-17 at 2 00 11 pm](https://user-images.githubusercontent.com/1254810/37559002-9d4b4bc2-29eb-11e8-94d2-978f528555ee.png)

**Comment by ahdigital** [comment]
We can use the input group component from ANT design to have a typeahead for the currencies inline with the quantity field.

![image](https://user-images.githubusercontent.com/3088454/37641606-5918cdc0-2c12-11e8-8623-81825191b641.png)

I removed the BS logo and reduced the amount of text from the modal header also.

**Comment by wmbutler** [comment]
Need this claimed to work now that UX is complete.

**Comment by ahdigital** [comment]
Bearing in mind that the ANT integration isn't yet complete, shouldn't we wait for development?

**Comment by wmbutler** [comment]
Sure, @gibbsfromncis which sprint should I place this in to coincide with ANT installation?

**Comment by gibbsfromncis** [comment]
@wmbutler please put this task to the next sprint. I suppose we should finish ant integration in 2 weeks

**Comment by ahdigital** [comment]
@wmbutler Can the UX for this be closed?

**Comment by wmbutler** [comment]
@ahdigital mind if I leave this for a later sprint rather than breaking it out? I'll just let the issue get worked here as well.

**Comment by wmbutler** [comment]
@calvinfroedge I see you claimed this. UX is done.

**Comment by wmbutler** [comment]
Related to #2142

**Comment by startailcoon** [comment]
Blocked by #2142

**Comment by startailcoon** [comment]
Fixed as part of #2205 and #2352 in PR #2517

**Comment by startailcoon** [comment]
@clockworkgr @sschiessl-bcp @ahdigital Please check that the UX bounty was paid out towards this issue as it wasn't closed until today.

No implementation bounty required for this as it's part of another issue.
